Plans/SwiftUITestSample_Crash.xctestplan

```diff
@@ -0,0 +1,28 @@
+{
+  "configurations" : [
+    {
+      "id" : "D5A59117-63AC-45A7-9579-3E6C2F6A84D5",
+      "name" : "Configuration 1",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "uiTestingScreenshotsLifetime" : "keepAlways"
+  },
+  "testTargets" : [
+    {
+      "skippedTests" : [
+        "EnvelopeTest",
+        "EnvelopeTest\/testCorruptedEnvelope()"
```

Comment:
I don't love this, it's not going to scale well. We'll have to add every test case to every test plan to skip, which IMO should only be used for tests that are not functioning correctly for some reason. Could we split them into separate Xcode UI test targets instead? Then they just get automatically included without having to mess with this extra layer of indirection.